packages/design-system/src/components/Promo/Promo.js

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import CloseIcon from 'react-material-icon-svg/dist/CloseIcon';
+
+import getMergedClassNames from '../../utils/getMergedClassNames';
+
+import { Button } from '../Button';
+import styles from './style.scss';
+
+const cx = classNames.bind(styles);
+
+const baseClass = 'promo';
+
+export const Promo = props => {
+  const {
+    buttonText,
+    children,
+    className,
+    header,
+    image,
+    light,
+    linkText,
+    onButtonClick,
+    onClose,
+    onLinkClick,
+    size,
+    img,
+    ...restProps
+  } = props;
+
+  const mergedWrapperClassNames = getMergedClassNames(
+    cx({
+      [`${baseClass}--light`]: light,
```

Comment:
Maybe we should consider some unification in theming. We [use](https://github.com/livechat/design-system/blob/develop/packages/design-system/src/components/TooltipNew/PopperTooltip.js#L404) a `theme` prop for `PopperTooltip`, while here we use `light` prop. I think we can use `light` for now, but discuss it during the next meeting.

Reply:
I would rather not use `theme` property name. It could be useful for actual theming. Your `kind` property is much better for that. Unifying it might be a good idea but IMHO it's not required until the third kind of Promo appears.